content/en/device-provisioning-service/client-library.md

```diff
@@ -186,6 +205,30 @@ dps_status_handler(plgd_dps_context_t *ctx, plgd_dps_status_t status, void *data
 }
 ```
 
+### DPS configuration resource
+
+To expose the `/plgd/dps` resource call `plgd_dps_set_configuration_resource(plgd_dps_context_t *ctx, bool create)`, with the `create` parameter equal to `true`.
```

Comment:
what is plgd/dps resource used for?
Is it published? Is it updated - does it reflect actual state? Who can write to it? When can I write to it? Do I need to own the device first? Do I need to disown the device after setting it? What if provisioning using endpoint provided here fails because I did a mistake? Can I still access the resource?

Reply:
What is a purpose of create parameter? what if it is set to false? 
you combine two different application logic into one function by using a create parameter. It's not clear to me why it's call set if its used for creation, and what setting without creation means. @jkralik could you please clarify this api?

Reply:
Could you please also explain what exactly is it used for?

The DPS resource gives the client possibility to trigger Zero Touch provisioning through the resource instead of C++ API. Additionally, this resource is published to the plgd hub which enables you to restart the provisioning process in case of infrastructure changes or other operation related use-cases.